src/test/java/com/expediagroup/hiveberg/TestInputFormatWithHadoopCatalog.java

```diff
@@ -165,14 +165,6 @@ public void testGetRecordReader() throws IOException {
     assertEquals(3, records.size() );
   }
 
-  @Test(expected = IllegalArgumentException.class)
```

Comment:
Is this test covered in whatever is testing `pathAsURI`?

Reply:
These 2 tests are covering the same bit of code I believe, but I'd consider these 2 tests highlighted here as more integration tests. I.e. we're checking that Hive is correctly setting the properties we're expecting via TBLPROPERTIES